third_party/blink/renderer/brave_font_whitelist.cc

```diff
@@ -758,6 +805,18 @@ const base::flat_set<base::StringPiece>& GetAllowedFontFamilies() {
   return kAllowedFontFamilies;
 }
 
+bool AllowFontByFamilyName(const AtomicString& family_name) {
+  std::string lower_ascii_name = family_name.LowerASCII().Ascii();
+  if (GetAllowedFontFamilies().contains(lower_ascii_name))
+    return true;
+#if BUILDFLAG(IS_ANDROID)
+  if (family_name.StartsWithIgnoringASCIICase("noto sans ") ||
```

Comment:
can we move this logic/guard inside `GetAllowedFontFamilies`?

Reply:
actually I'm a bit confused here, we have kAllowedFontFamilies for android for GetAllowedFontFamilies, but we're doing something different here? Can you add some comments to explain?

Reply:
Ah, actually `GetAllowedFontFamilies` is now only used for tests, so I will rename it with a `_for_testing` suffix and reorganize